app/Controllers/importExportController.php

```diff
@@ -38,9 +38,15 @@ public function indexAction(): void {
 	 */
 	private static function megabytes(string $size_str) {
 		switch (substr($size_str, -1)) {
-			case 'M': case 'm': return (int)$size_str;
-			case 'K': case 'k': return (int)$size_str / 1024;
-			case 'G': case 'g': return (int)$size_str * 1024;
+			case 'M':
```

Comment:
I personally prefer indented case inside switch statements. But when I set up the rules I tried to make them as close to the existing style as possible, not to add in my own opinion. ;-)

Reply:
Yes, me too, and it is also more common to have indented cases in switches, I believe